kibble/setup/kibble.ini

```
@@ -0,0 +1,18 @@
+[accounts]
+allowSignup = True
+verify = True
+
+[api]
+database = 2
+version = 0.1.0
+
+[elasticsearch]
+dbname = kibble
+host = elasticsearch
+port = 9200
+ss = False
+
+[mail]
+mailhost = localhost
+mailport = 25
```

```suggestion
mailhost = localhost:25
```
Probably we can consolidate here too